### ISSUE TITLE
lower default difficulty of bots

### ISSUE THREAD

**Issue by Gireen** [Unvanquished/Unvanquished#3055]
I think bots are to strong for new players, act to aggressive and can easily overwhelm newbies. 
It would be reasonable to lower the difficulty of bots in the default settings, since they are the first thing new players encounter offline or on many servers. 
Regulars play on servers with modified or tweaked bots so they should not be affected so much by it.

**Comment by DolceTriade** [comment]
I think we should make chaingun a bot skill and only grant it at higher levels?

**Comment by Gireen** [comment]
Sure its more work, but if someone wants to implement this it would be better.
The next release is somewhat in sight so if no one makes a skill until then i think its better to disable the chaingun and enable it again with a better solution. 

But the problem here is the overall difficulty not just the chaingun, which had recently the loudest complaints. 
There are many potential ways to make bots easier and i think we should simply try some and see how well they work. 
Sweet mentioned also disabling the Tyrant, that would be easy.
Or to have a max percentage of strong units of bots per team, that would be more complicated.

We could also lower the default skill level of 5.

**Comment by slipher** [comment]
> We could also lower the default skill level of 5.

Yeah that's what I was thinking. I don't like the idea of removing chaingun.
